frontend/src/pages/Admin/UserInfo.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Label, TextInput, Grid, Fieldset,
+} from '@trussworks/react-uswds';
+
+import RegionDropdown from '../../components/RegionDropdown';
+import JobTitleDropdown from '../../components/JobTitleDropdown';
+
+/**
+ * This component is the top half of the UserSection on the admin page. It displays and allows
+ * editing of basic user information.
+ */
+function UserInfo({ user, onUserChange }) {
+  return (
+    <Fieldset className="margin-bottom-2" legend="User Info">
+      <Grid row gap>
+        <Grid col={12}>
+          <Label htmlFor="input-email-name">Email</Label>
+          <TextInput id="input-email-name" type="text" name="email" value={user.email || ''} onChange={onUserChange} />
+        </Grid>
+        <Grid col={12}>
+          <Label htmlFor="input-full-name">Full Name</Label>
+          <TextInput id="input-full-name" type="text" name="fullName" value={user.fullName || ''} onChange={onUserChange} />
+        </Grid>
+      </Grid>
+      <Grid row gap>
+        <Grid col={6}>
+          <RegionDropdown id="user-region" name="region" value={user.region} onChange={onUserChange} />
```

Comment:
I was thinking it would be helpful to know what region the user is "assigned" to in order to default views to that region. If a user typically (always?) works in a specific region (and we know it) we can default views (such as region dropdowns) to that specific region.

I'm still thinking about how to handle the central office users, maybe add a `central office` option?

Reply:
If we're going to keep this control, then I think a `central office` option is needed.

I can also think of at least one user who does have roles in multiple regions. And a lot of feds are both `Program Specialists` and `COR` for title. That doesn't mean we can't use these settings as defaults for their normal regions and roles (I like that idea), but it does mean that we can't rely on them for authorization purposes.

Reply:
oh yeah sorry I should have made it clear I wasn't planning on this field being used for authorization or anything other than a helper for the UI.